Packs/Ja3er/Integrations/Ja3er/Ja3er.py

```diff
@@ -0,0 +1,58 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+
+class Client(BaseClient):
+    def __init__(self, base_url, *args, **kwarg):
+        super().__init__(base_url, *args, **kwarg)
+
+
+def test_module(client: Client, **args) -> str:
+    uri = f'/json'
+    client._http_request('GET', uri)
+
+    return "ok"
+
+
+def ja3_search(client: Client, **args) -> CommandResults:
+    ja3 = args.get('JA3')
+    uri = f'search/{ja3}'
+    r = client._http_request('GET', uri)
+
+    results = CommandResults(
+        outputs_prefix="JA3",
+        outputs_key_field=f"JA3.hash",
+        outputs={ja3: r},
```

Comment:
This path (`JA3.hash`) doesn't exist in the output you’re passing to the `outputs` argument. Please fix.

Reply:
The key field should be the hash that was searched, so that if the results of the same search change the value is updated in context. Not sure if I need to set outputs_key_field="JA3" or outputs_key_field=f"JA3.{ja3}"

Reply:
setting `outputs_key_field="JA3"` will do the trick